x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -43,7 +44,7 @@ export const SessionView = ({
   const [selectedProcess, setSelectedProcess] = useState<Process | null>(null);
   const [searchQuery, setSearchQuery] = useState('');
   const [searchResults, setSearchResults] = useState<Process[] | null>(null);
-  const [displayOptions, setDisplayOptions] = useState<DisplayOptionsState>({
+  const [displayOptions, setDisplayOptions] = useLocalStorage('displayOptions', {
```

Comment:
Kibana has already one library with a utility for `useLocalStorage`, from the `react-use` library, which is a set of handy hooks, can you give it a try? 

```js
import useLocalStorage from 'react-use/lib/useLocalStorage';
import { DisplayOptionsState } from ['../../../common/types/session_view']();
//...

const [displayOptions, setDisplayOptions] = useLocalStorage<DisplayOptionsState>('displayOptions', {
		timestamp: true,
    verboseMode: true,
});
```